tensorboard/components/BUILD

```diff
@@ -38,13 +38,33 @@ tf_ts_library(
     ],
 )
 
+# A target for development only. By dropping 'compile', this skips the expensive
+# minification step.
+tf_js_binary(
+    name = "polymer3_lib_binary_no_shim_dev",
+    compile = False,
+    entry_point = ":polymer3_lib.ts",
+    deps = [":polymer3_ts_lib"],
+)
+
```

Comment:
This might belong in `tensorboard/webapp/dev_assets` maybe?

Reply:
Done.